lib/WasmReader/WasmByteCodeGenerator.h

```diff
@@ -159,6 +159,7 @@ namespace Wasm
         Js::WebAssemblyModule* GenerateModule();
         void GenerateFunctionHeader(uint32 index);
     private:
+        Js::FunctionBody* GetFunctionBody() const { return nullptr; }
```

Comment:
this seems bad

Reply:
Agreed, I think there is only one case that needs it for the macro, so I'll just rewrite that part accordingly